OpenFoodFactsPower.user.js

```diff
@@ -1977,12 +2001,87 @@ ul#products_match_all > li > a > span { display: table-cell; width:   70%;  vert
         });
     }
 
-
     function hideListBarcodes() {
         $("svg.list_barcode").remove();
         $('ul[id^="products_"].search_results .with_barcode').removeClass('with_barcode');
     }
 
+    //shows HungerGames logo, rotate buttons
+    function showListButtons(){
+        let languageCode = getSubdomainLanguageCode();
+
+        $("ul[id^='products_'].search_results li[data-code]").each(function(index, element) {
+            let barcode = $(this).attr('data-code');
+            $(this).append('<a class="list_hunger_games_logo_search" alt="Hunger games logo search" title="Hunger games logo search" href="https://hunger.openfoodfacts.org/logos/search?barcode='+barcode+'"><span class="material-icons">image_search</span></a>');
+            $(this).append('<a class="list_rotate_image_270" alt="Rotate at 270°" title="Rotate at 270°"><span class="material-icons" style="transform: rotate(-90deg);">rotate_left</span></a>');
+            $(this).append('<a class="list_rotate_image_180" alt="Rotate at 180°" title="Rotate at 180°"><span class="material-icons">rotate_left</span></a>');
+            $(this).append('<a class="list_rotate_image_90" alt="Rotate at 90°" title="Rotate at 90°"><span class="material-icons" style="transform: rotate(90deg);">rotate_right</span></a>'); 
```

Comment:
```suggestion
            $(this).append('<a class="list_rotate_image_90" alt="Rotate at 90°" title="Rotate at 90°"><span class="material-icons">redo</span></a>');
```

Reply:
Here is the result:
![image](https://github.com/openfoodfacts/power-user-script/assets/562352/c03ae6cd-caa7-4c7c-879a-36a5b17ca9f3)